Week7/app/src/main/java/com/kulloveth/covid19virustracker/ui/status/DetailsFragment.kt

```diff
@@ -1,33 +1,35 @@
 package com.kulloveth.covid19virustracker.ui.status
 
+import android.content.res.TypedArray
```

Comment:
This is extremely cool. I wish it was working but as I search the web more I am thinking that perhaps the api is down because it's not working from some other pages that utilize the same api. I really think it's awesome that you made a graph and would love to see a screenshot, although, correct me if I'm wrong but I seem to remember this working in the last version.

Reply:
![graph](https://user-images.githubusercontent.com/25450066/88449265-5c97ca80-ce3d-11ea-8114-c004a268cab2.gif)

:) its just a basic bar chart as this is my first time trying it out, I hope more advanced ones will become easier when necessary